httpev.ml

```diff
@@ -382,10 +382,10 @@ let set_blocking req =
   req.blocking <- Some io;
   io
 
-let make_request_headers code hdrs =
+let make_request_headers ?version code hdrs =
```

Comment:
here also should not be optional

Reply:
yeah, agree. but there's 2 cases that I'm not sure if this PR is miss-leading: `send_reply_async` and `send_reply_blocking`. I would keep 1.0 on both cases to avoid those kinds of things https://github.com/ahrefs/devkit/pull/43#discussion_r2039264205